M/METIS@4/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+using BinaryBuilder
+
+name = "METIS"
```

Comment:
I think this needs to be `METIS4` so as to not overwrite the METIS_jll package which is at v5.

Reply:
Ideally it would actually be a v4.x tag of the METIS_jll package, right? So it wouldn't replace anything, just register an earlier version of the same thing. Not sure what would need to be done differently for that to happen.

Reply:
I followed the example set by GSL.

Reply:
According to Mosè below, what you've done accomplishes what I described anyway, so you've done it correctly it seems. :)